### PR TITLE
Save candidate data

### PR DESCRIPTION
**Pull request by warrensearle**
Saves candidate personal details. Tests passing.

### CODE REVIEW

File: src/views/Apply/AccountProfile/PersonalDetails.vue [hunk 1]
```diff
@@ -57,17 +57,20 @@ export default {
...
   data(){
+    const candidate = this.$store.getters['candidate/data']();
     return {
-      fullName: null,
-      email: null,
-      dateOfBirth: null,
-      nationalInsuranceNumber: null,
-
+      candidate: {
+        fullName: candidate && candidate.fullName || null,
```
Comment (LaurenQurashi) [inline src/views/Apply/AccountProfile/PersonalDetails.vue]: Is the ' candidate && candidate.fullName' part to check that to document exists in the first place?

Comment (warrensearle) [reply src/views/Apply/AccountProfile/PersonalDetails.vue]: Sure is. I was following pattern from `apply-admin` however in this instance `candidate` may not exist.
We may well amend the way we do this in the future, but works for now.

Comment (LaurenQurashi) [reply src/views/Apply/AccountProfile/PersonalDetails.vue]: That's smart I like that 😎

File: src/store.js [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
 import Vuex from 'vuex';
-import auth from '@/store/auth';
-
+import { vuexfireMutations } from 'vuexfire';
 Vue.use(Vuex);
 
+// Vuex modules
+import auth from '@/store/auth';
+import exercise from '@/store/exercise';
+import candidate from '@/store/candidate';
```
Comment (2-u) [inline src/store.js]: I think these are the tidbits I need over on the apply-admin Vue app! Thanks for exposing them.

**Pull request merged**